config/clock_deployment.yml

```diff
@@ -39,10 +39,10 @@ spec:
           resources:
             requests:
               cpu: 300m
-              memory: 300Mi
+              memory: 500Mi
             limits:
               cpu: 1000m
-              memory: 1Gi
+              memory: 4Gi
```

Comment:
The Clock performs a periodic bulk sync of apps and CF tasks that are running on the platform. While doing this it loads information about all the running LRPs into memory to compare against what's in CCDB. This can be pretty memory intensive on large environments so we should be more generous here.